Ant support library build broken

Compilation error in IDE running `eclipse.platform/resources/bundles/org.eclipse.core.resources/scripts/buildExtraJAR.xml`

Caused by 26abe910ee70ab5c1f74d4ba620ea039478531b9.

```
       [javac] ----------
       [javac] 1. ERROR in /data/git/eclipse.platform/resources/bundles/org.eclipse.core.resources/src_ant/org/eclipse/core/resources/ant/ConvertPath.java (at line 144)
       [javac] 	fileSystemPath = IPath.fromOSString(value.toString());
       [javac] 	                       ^^^^^^^^^^^^
       [javac] References to interface static methods are allowed only at source level 1.8 or above
       [javac] ----------
       [javac] 2. ERROR in /data/git/eclipse.platform/resources/bundles/org.eclipse.core.resources/src_ant/org/eclipse/core/resources/ant/ConvertPath.java (at line 155)
       [javac] 	resourcePath = IPath.fromOSString(value);
       [javac] 	                     ^^^^^^^^^^^^
       [javac] References to interface static methods are allowed only at source level 1.8 or above
       [javac] ----------
       [javac] ----------
       [javac] 3. ERROR in /data/git/eclipse.platform/resources/bundles/org.eclipse.core.resources/src_ant/org/eclipse/core/resources/ant/RefreshLocalTask.java (at line 113)
       [javac] 	IPath path = IPath.fromOSString(value);
       [javac] 	                   ^^^^^^^^^^^^
       [javac] References to interface static methods are allowed only at source level 1.8 or above
       [javac] ----------
       [javac] 3 problems (3 errors)

BUILD FAILED
/data/git/eclipse.platform/resources/bundles/org.eclipse.core.resources/scripts/buildExtraJAR.xml:38: Compile failed; see the compiler error output for details.
```

--- a/resources/bundles/org.eclipse.core.resources/scripts/buildExtraJAR.xml
+++ b/resources/bundles/org.eclipse.core.resources/scripts/buildExtraJAR.xml
@@ -35,7 +35,7 @@
 		<delete dir="${temp.folder}/ant_tasks/resources-ant.jar.bin"/>
 		<mkdir dir="${temp.folder}/ant_tasks/resources-ant.jar.bin"/>
 		<!-- compile the source code -->
-		<javac destdir="${temp.folder}/ant_tasks/resources-ant.jar.bin" failonerror="true" verbose="true" debug="on" target="1.5" source="1.5">
+		<javac destdir="${temp.folder}/ant_tasks/resources-ant.jar.bin" failonerror="true" verbose="true" debug="on" target="11" source="11">
 			<compilerarg value="-warn:-unavoidableGenericProblems"/>
 			<classpath>
 				<dirset dir="${basedir}/..">

